zirgen/dsl/driver.cpp

```diff
@@ -527,7 +538,7 @@ int main(int argc, char* argv[]) {
   pm.clear();
   // TODO: HoistAllocs is failing
   // pm.addPass(zirgen::Zhlt::createHoistAllocsPass());
-  pm.addPass(zirgen::ZStruct::createOptimizeLayoutPass());
+  // pm.addPass(zirgen::ZStruct::createOptimizeLayoutPass());
```

Comment:
was this intentional, or just for debugging? if intentional, perhaps the line should be removed instead of commented

Reply:
It was intentional.  Effectively, this pass causes some of the layout for the v2 to be discontinuous, which adds complexity to the preflight, but doesn't seem to make much improvement overall (# of the columns are the same).  That said, if we used the layout output properly in the preflight, this would not be an issue, so it may make sense to re-enable the pass once that is done.  I'll add a comment to clarify.